src/load_csv.py

```diff
@@ -16,20 +16,33 @@ def load_csv_and_generate_embeddings(path):
 
     df = df[necessary_cols]
 
+    # Create primary key column using category, subcategory, and question
+    df["primary_key"] = df.apply(lambda row: hashlib.sha256(
+        (
+            row["category"] + row["subcategory"] + row["question"]
+        ).encode()).hexdigest(), axis=1)
```

Comment:
`sha256` is not overkill for this use case? :thinking:

Reply:
@walison17 whats your suggestion? Sha256 makes it not collide easily and also 256 bits is not an enormous number.

Reply:
Done!